api/tests/scripts/beneficiary/import_test_users_test.py

```diff
@@ -23,6 +24,13 @@
 Pro,Pierre,pro@example.com,0123456789,06,06000,2000-01-01,PRO,11122233,PierrePro$123,interne:test
 """
 
+BOUNTY_EMAIL = "hackerù@bounty.ninja"
```

Comment:
Suggestion : tester avec un email assez long, correspondant à ce qu'on a réellement, car on ne sait jamais, si ça dépasse une taille max on va passer à côté dans les tests : 
```suggestion
BOUNTY_EMAIL = "unit-test-bug-bounty-hunter-0123456789abcdef@bugbounty.ninja"
```